docs/examples/rx/06_make_transfer/package.json

```diff
@@ -10,15 +10,7 @@
     "start": "node index.js"
   },
   "dependencies": {
-    "@cennznet/api": "^1.0.3",
-    "@cennznet/wallet": "^1.0.3"
-  },
-  "resolutions": {
```

Comment:
I think the resolutions are still relevant?

Reply:
Yup right, it slipped in from the original cherrypick. I have restored it. Thank you.